src/test/java/org/eolang/speco/MainTest.java

```diff
@@ -57,44 +59,53 @@ public final class MainTest {
      */
     private final Path eos = this.tests.resolve("eo");
 
-    @Test
-    public void convertsFromXmir(@TempDir final Path temp) throws IOException {
-        final Path output = this.runSpeco(false, temp);
-        for (final Path path : Files.newDirectoryStream(output)) {
-            final List<String> expected = Files.readAllLines(path);
-            final List<String> produced = Files.readAllLines(temp.resolve(path.getFileName()));
-            Assertions.assertEquals(expected, produced);
-        }
+    @ParameterizedTest
+    @ValueSource(strings = {"simple"})
+    public void convertsFromXmir(final String name, @TempDir final Path temp) throws IOException {
+        final Path output = this.runSpeco(this.xmirs.resolve(name), temp, false);
+        this.compare(temp, output);
     }
 
-    @Test
-    public void convertsFromEo(@TempDir final Path temp) throws IOException {
-        final Path output = this.runSpeco(true, temp);
-        for (final Path path : Files.newDirectoryStream(output)) {
+    @ParameterizedTest
+    @ValueSource(strings = {"booms", "pets"})
+    public void convertsFromEo(final String name, @TempDir final Path temp) throws IOException {
```

Comment:
@advasileva Probably you can inline `output` and `produced` variables.
Please read this article: https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html